frontend/app/src/components/settings/accounting/rule/AccountingRuleSetting.vue

```diff
@@ -204,45 +155,120 @@ function showDeleteConfirmation(item: AccountingRuleEntry) {
   );
 }
 
-function getType(eventType: string, eventSubtype: string) {
-  return get(
-    getEventTypeData({
-      eventSubtype,
-      eventType,
-    }),
-  );
+async function handleRuleAction(action: 'add-general' | 'add-event-specific' | 'edit-general' | 'edit-event-specific') {
```

Comment:
If I am not mistaken we use this elsewhere, can we do a const, type alias and use the const values and the type instead?